Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/ContentTypeSnippetTests.cs

```diff
@@ -0,0 +1,177 @@
+        {
+            var client = _fileSystemFixture.CreateMockClientWithResponse("Snippet.json");
+
+            var expected = _fileSystemFixture.GetExpectedResponse<ContentTypeSnippetModel>("Snippet.json");
+
+            List<ContentTypeSnippetOperationBaseModel> changes = new()
+            {
+                new ContentTypeSnippetPatchRemoveModel
+                {
+                    Path = $"/elements/codename:none"
+                },
+                new ContentTypeSnippetPatchReplaceModel
+                {
+                    Value = "Provide all personas for which this article is relevant.",
+                    Path = $"/elements/codename:personas/guidelines"
+                },
+                new ContentTypeSnippetAddIntoPatchModel
+                {
+                    Value = new TextElementMetadataModel
+                    {
+                        Name = "Meta description",
+                        Guidelines = "Sum up the blog for SEO purposes. Limit for the meta description is 160 characters.",
+                        IsRequired = false,
+                        ExternalId = "b9dc537c-2518-e4f5-8325-ce4fce26171e",
+                        Codename = "meta_description",
+                        MaximumTextLength = null
+                    },
+                    After = Reference.ByCodename("personas"),
+                    Path = "/elements"
+                }
+            };
+
+            var response = await client.ModifyContentTypeSnippetAsync(Reference.ByCodename("metadata"), changes);
+
+            response.Should().BeEquivalentTo(expected);
+        }
+
+        [Fact]
+        public async void ModifyContentTypeSnippetAsync_IdentifierIsNull_ModifiesContentType()
+        {
+            var client = _fileSystemFixture.CreateMockClientWithResponse("Snippet.json");
+
+            var expected = _fileSystemFixture.GetExpectedResponse<ContentTypeSnippetModel>("Snippet.json");
+
+            List<ContentTypeSnippetOperationBaseModel> changes = new() {
+                new ContentTypeSnippetPatchRemoveModel
+                {
+                    Path = $"/elements/codename:none"
+                }
+            };
+
+            await client.Invoking(x => x.ModifyContentTypeSnippetAsync(null, changes)).Should().ThrowAsync<ArgumentNullException>();
+        }
+
+        [Fact]
+        public async void ModifyContentTypeSnippetAsync_ChangesAreNull_ModifiesContentType()
+        {
+            var client = _fileSystemFixture.CreateMockClientWithResponse("Snippet.json");
+
+            var expected = _fileSystemFixture.GetExpectedResponse<ContentTypeSnippetModel>("Snippet.json");
+
+            await client.Invoking(x => x.ModifyContentTypeSnippetAsync(Reference.ByCodename("metadata"), null)).Should().ThrowAsync<ArgumentException>();
+        }
+
+        [Fact]
+        public async void ModifyContentTypeSnippetAsync_NoChanges_ModifiesContentType()
```

Comment:
test name is _ModifyContentTypeSnippetAsync_NoChanges_ModifiesContentType_ but actually checks throwing an exception. There are also other places with the same issue.

Reply:
There were also naming problems in ContentType tests. Fixed it all.